api/scripts/add-or-update-certification-centers-dpo-infos.js

```diff
@@ -1,96 +1,59 @@
-import 'dotenv/config';
+import Joi from 'joi';
 
-import * as url from 'node:url';
-
-import _ from 'lodash';
-
-import { disconnect } from '../db/knex-database-connection.js';
 import { updateCertificationCenterDataProtectionOfficerInformation } from '../lib/domain/usecases/update-certification-center-data-protection-officer-information.js';
 import * as dataProtectionOfficerRepository from '../lib/infrastructure/repositories/data-protection-officer-repository.js';
-import { checkCsvHeader, parseCsvWithHeader } from './helpers/csvHelpers.js';
-
-const modulePath = url.fileURLToPath(import.meta.url);
-const IS_LAUNCHED_FROM_CLI = process.argv[1] === modulePath;
-const REQUIRED_FIELD_NAMES = ['certificationCenterId', 'firstName', 'lastName', 'email'];
+import { BaseScript } from '../src/shared/application/scripts/base-script.js';
+import { csvFileParser } from '../src/shared/application/scripts/parsers.js';
+import { ScriptRunner } from '../src/shared/application/scripts/script-runner.js';
+
+const columnsSchemas = [
+  { name: 'certificationCenterId', schema: Joi.number() },
+  { name: 'firstName', schema: Joi.string().trim().optional() },
+  { name: 'lastName', schema: Joi.string().trim().optional() },
+  { name: 'email', schema: Joi.string().trim().replace(' ', '').lowercase().email().optional().default(null) },
+];
+
+export class AddOrUpdateCertificationCentersDpoInfosScript extends BaseScript {
```

Comment:
Dans le code en anglais on écrit plutôt `info` au singulier : 
```suggestion
export class AddOrUpdateCertificationCentersDpoInfoScript extends BaseScript {
```